gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py

```diff
@@ -130,14 +136,19 @@ def AuthenticateOutOfBand(self, requestParameters):
         if enteredCode == identity.getWorkingParameter("oobCode"):
             facesMessages.clear()
             updateNeeded = False
+            telemetry["result"] = "sucess"
 
             if contact is not None: # Registration
+                telemetry["step"] = "code verification"
```

Comment:
isn't this a step for both oob registration and auth?

Reply:
Yes. Autentication is tracked [here](https://github.com/sign-in-canada/Acceptance-Platform/blob/ebdb3eeb376670782dd4b84b5143b459028957ae/gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py#L151).

Reply:
I meant shouldn't be outside the if statement (i.e injected in line 140)